validation/src/main/scala/hmda/validation/engine/lar/syntactical/LarSyntacticalEngine.scala

```diff
@@ -1,19 +1,29 @@
 package hmda.validation.engine.lar.syntactical
 
 import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.model.institution.Institution
 import hmda.validation.api.ValidationApi
+import hmda.validation.context.ValidationContext
 import hmda.validation.engine.lar.LarCommonEngine
+import hmda.validation.rules.ts.syntactical.S025
 import hmda.validation.rules.lar.syntactical._
 
 trait LarSyntacticalEngine extends LarCommonEngine with ValidationApi {
 
-  def checkSyntactical(lar: LoanApplicationRegister): LarValidation = {
+  private def s025(lar: LoanApplicationRegister, ctx: ValidationContext): LarValidation = {
+    convertResult(lar, S025(lar, ctx), "S025")
+  }
+
+  def checkSyntactical(lar: LoanApplicationRegister, ctx: ValidationContext): LarValidation = {
     val checks = List(
       S010,
       S020,
       S205
     ).map(check(_, lar))
 
+    // Exclude edits requiring institution data
+    if (ctx.institution.isDefined) checks :+ s025(lar, ctx)
```

Comment:
This bug has been fixed.  In addition, I've made a very similar fix (https://github.com/cfpb/hmda-platform/pull/452/commits/f4ecec7818c1c6412069c37f422a3ce873b62f0c) to `LarQualityEngine`, which I borrowing to offending logic from, and was itself not working as expected.  Doh!  Classic example of copy/paste programming.  Let that be a lesson to me.

That said, this got past me primarily because I couldn't figure out a good way to test it.  There isn't currently (as far as I can tell) a clean way to get a list of which edits actually ran in a given "engine", so it's hard to write that class of tests.  The current option is to stage a given edit to fail, and if engine then fails, you can assume the edit was run.

We're not going to try to solve this on this PR.  I'll create separate issue for further discussion.

Again, good catch @lgdean!